### DISCUSSION TITLE
My Dream: Nota = Roam Research + Obsidian love child

### DISCUSSION THREAD

**Discussion by NoodleNodes** [notaapp/talk#276] [Feature Requests]
I no longer use Obsidian… I feel overwhelmed by all the plugins and tinkering to make it feel just right. I’m moving away from Roam Research as I miss the control of data and folders. Nota is the first app in a long while that has just clicked for me… the biggest sell points, a beautiful clean markdown environment (well done team!) and keyboard navigation!

I am a writer and an outliner, for me there are benefits to both strategies for professional writing. A flat markdown page for prose, an outline for thinking. 

I would love to see Nota fill this gap by delivering a well-rounded tool for any writer and thinker. Where bullets become individual blocks that can be focussed into and linked to (Roam Research, LogSeq), and markdown pages can be referenced to the line or paragraph (Obsidian).

Making linked and unlinked references more prominent on the page would also be beneficial. Maybe the user can decide in the preferences to choose between minimal and expanded views to suit their needs?

It’s a big ask, but please consider this for future development and updates.

keep doing what you’re doing! Very excited to see what you have planned for the writing and thinking space 🥰👍

**Comment by astoilkov** [comment]
> I no longer use Obsidian… I feel overwhelmed by all the plugins and tinkering to make it feel just right. I’m moving away from Roam Research as I miss the control of data and folders. Nota is the first app in a long while that has just clicked for me… the biggest sell points, a beautiful clean markdown environment (well done team!) and keyboard navigation!

❤️ ❤️ ❤️

> I would love to see Nota fill this gap by delivering a well-rounded tool for any writer and thinker. Where bullets become individual blocks that can be focussed into and linked to (Roam Research, LogSeq), and markdown pages can be referenced to the line or paragraph (Obsidian).

Do you know of such an application? Did you always use two apps for each?

What about being able to link to bullets? This way you can mindfully force outline in your documents. They will consist of bullets only and you will link to other bullets.

> Making linked and unlinked references more prominent on the page would also be beneficial. Maybe the user can decide in the preferences to choose between minimal and expanded views to suit their needs?

I like that idea. We are currently experimenting with showing more contextual references at the end of each document. Also, we are bringing references to the right sidebar soon.

**Comment by NoodleNodes** [comment]
> Do you know of such an application?

Not that I can think of... you'd be the first to create such a love-child 😉

> Did you always use two apps for each?

Yes, and it's not a very productive way of working with some notes in one app and others in another. Ideally, all writing and note-taking/thinking should be centralized, easily cross-referenceable and searchable.

> What about being able to link to bullets? This way you can mindfully force outline in your documents. They will consist of bullets only and you will link to other bullets.

You read my mind 🥰 I think this is the basic premise of Roam Research and Loqseq, but in a standard markdown writing app, bullets would be manually created yes, and those bullets would then become linkable and perform the same function as those found in Roam Research and Logseq. For longer form pieces (walls of text as paragraphs and sentences) the ability to link to these links like in Obsidian would be great. I'm currently doing this in a hacky manual way by creating `([[^17fdea]])` references at the two points in text I want to reference, or ([[annotation^f72066]]) for comments to passages in ebooks converted to markdown. 

If you guys haven't yet, please do play around a little with the Roam Research and Loqseq interfaces and features, there are some that would be greatly appreciated in Nota. I'm going for broke here... but an outline panel where one can drag and drop paragraph blocks or bullets, would be a wonderfully productive addition as it helps the professional writer outline and sort and re-arrange the structure of the piece before committing to writing. Also, clicking on a bullet will 'focus' into that bullet for a cleaner writing experience would be wonderful! 

> I like that idea. We are currently experimenting with showing more contextual references at the end of each document. Also, we are bringing references to the right sidebar soon.

Side bar sounds good... but you may wish to consider an 'inline to the page' approach and save that real-estate for other writing dedicated features (cheeky glance up at the Outline suggestion above). What I love about what you have done with the Page Previews, you can edit them while in a note! One of my favorite features and I brought the license immediately upon discovering this! I'd like to be able to do the same with linked references below an active note. The benefit of keeping everything inline on the page is that relevant links are not hidden or subjected to a secondary window view that I have to click into to view. 'Out of mind, out of sight' and all. They are up-front and present in the working note. But just my two-cents 🤓🤪

Thanks for the reply. You guys are doing a great job! Keep driving innovation and don't be scared to copy or steal from the best, just remember to make it your own 😉 

> Pablo Picasso on Creativity, “Good artists copy, great artists steal.”

**Reply by astoilkov** [reply]
Thanks for all the feedback. All greatly appreciated.

We will consider everything. Hope you stay with us for a long time.
